gradle/scripts/lib/kotlin.gradle

```diff
@@ -36,9 +36,23 @@ configure(projectsWithFlags('kotlin')) {
         }
 
         afterEvaluate {
-            // A workaround for 'runKtlintCheckOverMainSourceSet' uses this output of task ':generateProto' without
-            // declaring an explicit or implicit dependency.
-            tasks.runKtlintCheckOverMainSourceSet.dependsOn(project.ext.getGenerateSourcesTask())
+
+            // `tasks.withType(KtLintCheckTask)` does not work here.
+            ['Main', 'Test', 'Jmh'].each { name ->
```

Comment:
`jmh` module injects `runKtlintCheckOverJmhSourceSet` and `compileJmhKotlin` that tasks need explicit dependencies for generated sources from proto or thrift files.